common/src/androidMain/kotlin/com/cuhacking/atlas/common/DataCache.kt

```diff
@@ -0,0 +1,28 @@
+package com.cuhacking.atlas.common
+
+import android.content.Context
+import kotlinx.coroutines.withContext
+import kotlinx.datetime.Clock
+import kotlinx.datetime.Instant
+import java.io.File
+
+actual class DataCache actual constructor(dispatchers: CoroutineDispatchers) {
+    lateinit var appContext: Context
+    actual var lastModified: Instant? = null
+    private val dispatchers = CoroutineDispatchers
```

Comment:
`dispatchers` can be made a constructor property.
```suggestion
actual class DataCache actual constructor(private val dispatchers: CoroutineDispatchers = CoroutineDispatchers) {
    lateinit var appContext: Context
    actual var lastModified: Instant? = null
```

Reply:
I tried to do this earlier, but it kept giving me an error `"Actual function cannot have default argument values, they should be declared in the expected function"`. However, I was able to suppress it and the tests ran successfully. I don't really know why this error came up, should I make the change? (same thing with the rest of the actual classes)

Reply:
Ah, in the expect class constructor you have to add the default argument. `expect class DataCache(dispatchers: CoroutineDispatchers = CoroutineDispatchers)`

Reply:
That was after adding it

Reply:
Oh huh 🤔 
Well, we don't need the default argument. We can just specify it when constructing it (even if it seems a little inconvenient).

Reply:
Well...suppressing it worked for some reason, but I don't know if it's going to cause issues later 😆